### ISSUE TITLE
Unable to match cartridge in debugger after VSCode update

### ISSUE THREAD

**Issue by BryanOttho** [SqrTT/prophet#135]
After updating to the latest version of VSCode 1.28.0 on OSX the debugger won't hit breakpoints and has the message: "Unable match cartridge: null"

Debug console screenshot attached:
![image](https://user-images.githubusercontent.com/43272360/46658482-195e6000-cbb3-11e8-9ae4-771dd44db032.png)

**Comment by sebbeyer** [comment]
I have the same problem but with error message: 

From client: threads(undefined)
From client: threads(undefined)
To client: {"seq":0,"type":"response","request_seq":9,"command":"threads","success":true,"body":{"threads":[{"id":10,"name":"thread 10"}]}}
From client: stackTrace({"threadId":10,"startFrame":0,"levels":20})
From client: stackTrace({"threadId":10,"startFrame":0,"levels":20})
To client: {"seq":0,"type":"response","request_seq":10,"command":"threads","success":true,"body":{"threads":[{"id":10,"name":"thread 10"}]}}
To client: {"seq":0,"type":"event","event":"output","body":{"category":"stderr","output":"Unable match cartridge: \"app_storefront_controllers/cartridge/scripts/guard.js\""}}
Unable match cartridge: "app_storefront_controllers/cartridge/scripts/guard.js"

**Comment by SqrTT** [comment]
@sebbeyer probably your `app_storefront_controllers` doesn't have a `.project` file that's why it is not recognized as cartridge

**Comment by SqrTT** [comment]
@BryanOttho thank you for reporting. Error message "Unable match cartridge: null" have nothing with the issue itself (message there was way ago it appears for a template if it shows in a stack trace).

Will investigate this further since not clear what caused the issue yet

**Comment by sebbeyer** [comment]
@SqrTT yes it does not have one like many others in our project that we never opened with Eclipse. So why should there be a .project file?

**Comment by SqrTT** [comment]
@sebbeyer 
Official tool to develop sites is Eclipse and project should work there.
This is the only way to detect the cartridge so extension relays on it.

**Comment by sebbeyer** [comment]
@SqrTT ok, yes I know Eclipse is the main platform. We try to avoid it. Works with the .project file now. Just thought that it is weird for a VS Code extension to relay on something eclipse produces. I guess you could also simply check the folder structure or the list of folders in the main /cartridges folder

**Comment by SqrTT** [comment]
@sebbeyer that was the first implementation but then appears that different people have a different structure too (first edge case is modules folder...)

**Comment by sebbeyer** [comment]
@SqrTT ok, thanks for explaining 💃
